webui/portal/src/main/java/org/exoplatform/portal/application/PortalRequestContext.java

```diff
@@ -295,8 +296,12 @@ public UserPortal getUserPortal() {
     }
 
     public UserPortalConfig getUserPortalConfig() {
+        String remoteUser = "";
         if (userPortalConfig == null) {
-            String remoteUser = getRemoteUser();
+            ConversationState conversationState = ConversationState.getCurrent();
+            if(conversationState != null) {
```

Comment:
Should check if user is anonymous IMO (even if it doesn't happen now, we may face an improvement that will change to have an "ANONIM" user when not connected, thus just in case...)

```suggestion
            if (conversationState != null
              && conversationState.getIdentity() != null
              && !IdentityConstants.ANONIM.equals(conversationState.getIdentity().getUserId())) {
```

Reply:
Yes that makes sense